Documentation/security/index.rst

```diff
@@ -4,12 +4,35 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. _security_root:
```

Comment:
There is something a bit strange about the flow of the Security section when it's build - there's a "front page" with the index and some introductory content about network policy, but then the next page is "Introduction"
<img width="1088" alt="Screenshot 2022-08-10 at 22 20 56" src="https://user-images.githubusercontent.com/458616/184022524-2a8bb4ed-a1e6-4a48-a418-49cd0ff04a2b.png">

Reply:
Resolved